pkg/rkeworker/execute_windows.go

```diff
@@ -0,0 +1,269 @@
+package rkeworker
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/rancher/pkg/rkecerts"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+var (
+	rancherDir = "C:\\etc\\rancher"
+)
+
+func ExecutePlan(ctx context.Context, nodeConfig *NodeConfig) error {
+	if nodeConfig.Certs != "" {
+		bundle, err := rkecerts.Unmarshal(nodeConfig.Certs)
+		if err != nil {
+			return err
+		}
+
+		if err := bundle.Explode(); err != nil {
```

Comment:
Did you change the cert location in here? i believe by default bundle.explore write certs to `/etc/kubernetes` which is not windows-friendly

Reply:
Windows writer will put those certs starting with `/` to the boot disk ( `c:`).